### ISSUE TITLE
Can't manage mods/maps with the FAF client

### ISSUE THREAD

**Issue by PhilipJFryFAF** [FAForever/client#732]
If your windows username has a mutated vovel in it eg "ö" then FAF cannot look up the mod folder
[forever.zip](https://github.com/FAForever/client/files/1000534/forever.zip)

Seems like the client stores the mods/maps in the programdata folder as a fallback.

The mods appear to be available in the game lobby though.

**Comment by PhilipJFryFAF** [comment]
asked the user in question to use the 0.12.5 client preview and the error still occurs there
[forever0.12.5preview.zip](https://github.com/FAForever/client/files/1000822/forever0.12.5preview.zip)

**Comment by duk3luk3** [comment]
Thanks for your report.

In the future please include the actual error output!

```
2017-05-15 12:32:55,762 INFO     root                           FAF version: 0.12.4-57-g7a4aff3+1810 Environment: production
2017-05-15 12:32:55,763 INFO     util                           PERSONAL_DIR initial: C:\Users\König\Documents
2017-05-15 12:32:55,763 ERROR    util                           PERSONAL_DIR not ok, falling back.
Traceback (most recent call last):
  File "src\util\__init__.py", line 101, in <module>
UnicodeEncodeError: 'ascii' codec can't encode character u'\xf6' in position 10: ordinal not in range(128)
```

**Comment by GrotheFAF** [comment]
Do we really need the ascii decode?
https://github.com/FAForever/client/blob/develop/src/util/__init__.py#L101
I removed the line and did some ö ß ä in my PERSONAL_DIR (debug at L98) and Modvault was working like before

**Comment by duk3luk3** [comment]
@PhilipJFryFAF What exactly is the problem here? If the username has a non-ascii character we fall back to using ProgramData.

We have to use ProgramData if the username has non-ascii characters because otherwise supcom itself will fail.

**Comment by PhilipJFryFAF** [comment]
the issue is that the user doesn't know where the mods/maps are and gets confused
think the user also couldn't manage select mods/maps in the client also

**Comment by GrotheFAF** [comment]
We could add the personal_dir (with mods/maps folder) to the Links in faf-menu, or maybe under Options - Settings

**Comment by PhilipJFryFAF** [comment]
that would be great
right now the "easiest" way to find mods/maps folder via qai commands

**Comment by GrotheFAF** [comment]
and qai commands only give the 'clean'-name path, not the nonascii-'dirty'-name case path.
still the question: where in menu?

**Comment by PhilipJFryFAF** [comment]
this error pops up when trying to manage ui mods
![image](https://cloud.githubusercontent.com/assets/24656673/26328765/9ba33470-3f44-11e7-8405-c49b5f709033.png)


sim mods can't be enabled via the prompt where you choose the lobby name/map/mods as well

**Comment by GrotheFAF** [comment]
you should do what it says ... add the forever.log (here)

**Comment by PhilipJFryFAF** [comment]
ok sorry for the confusion: it looks like the error in the screenshot is related to a user not having a game.prefs file - forgot to run the game before testing
once i made sure the user on the VM had a game.prefs file i was able to manage ui mods via the vault and select specific sim mods/a map in the prompt where you choose them

so the only issue that remains is a user being confused when the path for the mods/maps is not the one that qai tells them with the $modpath/$mappath command which will be addressed by #735

**Comment by PhilipJFryFAF** [comment]
seems like the error in the screenshot is related to not having a game.prefs file -> when having one the client does handle the mods/maps in the gamedata folder just fine

only issue is that the user will be directed to the wrong path via the $modpath/$mappath commands from qai but this will be fixed with #735

**Comment by duk3luk3** [comment]
OK, closing this for now.
